stats/release_queue.php

```diff
@@ -11,143 +11,263 @@
+
+    $user_can_see_queue_settings = $ordinary_users_can_see_queue_settings ||
+        user_is_a_sitemanager() || user_is_proj_facilitator();
+
     $sql = sprintf("
         SELECT *
         FROM queue_defns
         WHERE round_id='%s' AND name='%s'",
-        DPDatabase::escape($round_id),
+        DPDatabase::escape($round->id),
         DPDatabase::escape($name)
     );
     $qd = mysqli_fetch_object(DPDatabase::query($sql));
     if (!$qd) {
-        die(html_safe("No such release queue '$name' in $round_id."));
+        die(html_safe("No such release queue '$name' in $round->id."));
     }
-    $cooked_project_selector = cook_project_selector($qd->project_selector);
-    $comment = $qd->comment;
+
+    echo "<p><a href='?round_id=$round->id'>" . sprintf(_("Back to release queues for round %s"), $round->id) ."</a></p>\n";
 
     // TRANSLATORS: %s is the name of this release queue.
-    $title = sprintf(_("\"%s\" Release Queue"), $name);
-    output_header($title, NO_STATSBAR);
-    echo "<h1>" . html_safe($title) . "</h1>";
+    echo "<h2>" . html_safe(sprintf(_('%1$s release queue: %2$s'), $round->id, $name)) . "</h2>";
+
+    $cooked_project_selector = cook_project_selector($qd->project_selector);
+
+    [$length, $length_unheld] = _get_queue_length($cooked_project_selector, $round->project_waiting_state);
 
-    // Add Back to to Release Queues link
-    echo "<p><a href='".$code_url."/stats/release_queue.php?round_id=$round_id'>"._("Back to Release Queues")."</a></p>\n";
+    $fields = [
+        _("Comment") => $qd->comment,
+        _("Status") => $qd->enabled ? _("enabled") : _("disabled"),
+        _("Current length") => $length,
+        _("Length without holds") => $length_unheld,
+    ];
 
     if ($user_can_see_queue_settings) {
-        echo "<p><b>" . _("Selector") . ":</b> $qd->project_selector</p>\n";
+        $fields[_("Selector")] = $qd->project_selector;
         if ($cooked_project_selector != $qd->project_selector) {
-            echo "<p><b>" . _("Filled-in") . ":</b> $cooked_project_selector</p>\n";
+            $fields[_("Filled-in")] = $cooked_project_selector;
         }
-        echo "<p><b>" . _("Comment") . ":</b> $comment</p>\n";
+        $fields[_("Release Criterion")] = $qd->release_criterion;
+    }
+
+    echo "<p>";
+    foreach ($fields as $label => $value) {
+        echo "<b>" . html_safe($label) . "</b>: " . html_safe($value) . "<br>";
+    }
+    echo "</p>";
+
+    echo "<h3>" . _("Recent queue statistics") . "</h3>";
+
+    echo "<p>" . _("Number of projects and pages released from this queue recently.") . "</p>";
+
+    echo "<table class='themed theme_striped' style='width: auto;'>\n";
+    echo "<tr>";
+    echo "<th>" . _("Days ago") . "</th>";
+    echo "<th>" . _("Projects") . "</th>";
+    echo "<th>" . _("Pages") . "</th>";
+    echo "</tr>";
+
+    foreach ([1, 7, 28, 84] as $days_ago) {
+        $seconds_ago = time() - 60 * 60 * 24 * $days_ago;
```

Comment:
not that it is too taxing and I've probably wasted more cycles typing this than we'll ever save, but we could inline some of these computations: 
```
define("SECONDS_IN_DAY", 86400);
//...
time() - SECONDS_IN_DAY * $days_ago;
```